internal/provider/logs_export_resource.go

```diff
@@ -0,0 +1,360 @@
+				Attributes: map[string]schema.Attribute{
+					"bucket": schema.StringAttribute{
+						Required:            true,
+						MarkdownDescription: "Bucket.\nName of the S3 bucket to export logs to.",
+						PlanModifiers:       []planmodifier.String{stringplanmodifier.UseStateForUnknown()},
+					},
+					"bucket_layout": schema.StringAttribute{
+						Required:            true,
+						MarkdownDescription: "Folder name\nFolder where logs will be exported. \n Can include the date as a template variable in the Go date format, such as \"2006/01/02/some_folder\".",
+						PlanModifiers:       []planmodifier.String{stringplanmodifier.UseStateForUnknown()},
+					},
+					"aws_access_key_id": schema.StringAttribute{
+						Required:            false,
+						MarkdownDescription: "Access key ID",
+						PlanModifiers:       []planmodifier.String{stringplanmodifier.UseStateForUnknown()},
+					},
+					"aws_secret_access_key": schema.StringAttribute{
+						Required:            false,
+						MarkdownDescription: "Secret access key",
+						PlanModifiers:       []planmodifier.String{stringplanmodifier.UseStateForUnknown()},
+					},
+					"region": schema.StringAttribute{
+						Required:            true,
+						MarkdownDescription: "Region\nRegion where the bucket is located.",
+						PlanModifiers:       []planmodifier.String{stringplanmodifier.UseStateForUnknown()},
+					},
+					"endpoint": schema.StringAttribute{
+						Required:            true,
+						MarkdownDescription: "Endpoint\nEndpoint of the S3-compatible service. Leave blank if you're using AWS.",
+						PlanModifiers:       []planmodifier.String{stringplanmodifier.UseStateForUnknown()},
+					},
+					"skip_verify_ssl_cert": schema.BoolAttribute{
+						Optional:            true,
+						Computed:            true,
+						Default:             booldefault.StaticBool(false),
+						MarkdownDescription: "Skip verifying SSL certificate\nSelect if the bucket allows self-signed certificates.",
+						PlanModifiers: []planmodifier.Bool{
+							boolplanmodifier.RequiresReplace(),
+						},
+					},
+					"disable_ssl": schema.BoolAttribute{
+						Optional:            true,
+						Computed:            true,
+						Default:             booldefault.StaticBool(false),
+						MarkdownDescription: "Allow connections without SSL\nSelect if you're connecting to an S3-compatible service that doesn't use SSL/TLS.",
+						PlanModifiers: []planmodifier.Bool{
+							boolplanmodifier.RequiresReplace(),
+						},
+					},
+				},
+				Validators: []validator.Object{},
+			},
+			"datadog": schema.SingleNestedAttribute{
+				Optional:            true,
+				MarkdownDescription: "Datadog export target",
+				Attributes: map[string]schema.Attribute{
+					"api_key": schema.StringAttribute{
+						Required:            true,
+						MarkdownDescription: "API Key for Datadog",
+					},
+					"datadog_host": schema.StringAttribute{
+						Required:            true,
+						MarkdownDescription: "Host name for Datadog",
```

Comment:
```suggestion
						MarkdownDescription: "Datadog site. Make sure to specify the correct site because Datadog sites are independent and data isn't shared across them by default",
```